deeppavlov/dataset_readers/conll2003_reader.py

```diff
@@ -80,12 +81,20 @@ def parse_ner_file(self, file_name: Path):
                         tags = []
                 else:
                     if self.provide_pos:
-                        token, pos, *_, tag = line.split()
-                        pos_tags.append(pos)
+                        try:
+                            token, pos, *_, tag = line.split()
+                            pos_tags.append(pos)
+                        except:
+                            log.warning('Skip {}, splitted as {}'.format(repr(line), repr(line.split())))
```

Comment:
```suggestion
                            log.warning('Skip {}, splitted as {}'.format(repr(line), repr(line.split())))
                            continue
```
for both cases. Otherwise `token` and `pos` variables might be repeated from a previous iteration